.github/file-paths.yaml

```
@@ -82,6 +82,15 @@ sources: &sources
   - *frontend_sources
   - *backend_sources
 
+e2e_ci: &e2e_ci
+  - ".github/actions/build-e2e-matrix"
+  - ".github/actions/prepare-frontend/**"
+  - ".github/actions/prepare-backend/**"
```

Why could we not have `**` for other folders, but only these 2? Can't we do the same here?
```suggestion
  - ".github/actions/prepare-frontend"
  - ".github/actions/prepare-backend"
```

It seems all these folders contain a single file called `action.yml`, so I think the path could be the same.